installers/windows/nsis/oq-server.bat

```diff
@@ -13,8 +13,12 @@ if not exist python3\pycached (
    copy /y nul python3\pycached >nul
 )
 
-echo Starting the server.
-echo Please wait ...
+echo Obtaining OQ Engine version
+for /f %%i in ('oq -v') do set OQ_VERSION=%%i
```

Comment:
You are adding 5 seconds to an already long startup time; it is better to change the `oq webui start` command to print the version number instead.

Reply:
Agreed, I will modify the batch again